src/main/java/io/github/connorhartley/guardian/Guardian.java

```diff
@@ -62,7 +66,10 @@
         description = "An extensible anticheat plugin for Sponge.",
         authors = {
                 "Connor Hartley (vectrix)"
-        }
+        },
+        dependencies = @Dependency(
```

Comment:
The way it's setup it needs to be there. It'd require a fair few changes to be optional.